packages/financial-templates-lib/src/price-feed/UniswapPriceFeed.js

```diff
@@ -26,14 +39,26 @@ class UniswapPriceFeed extends PriceFeedInterface {
     this.getTime = getTime;
     this.historicalLookback = historicalLookback;
     this.invertPrice = invertPrice;
+    // The % of the lookback window (historicalLookback + twapLength) that we want to query for Uniswap
+    // Sync events. For example, 1.1 = 110% meaning that we'll look back 110% * (historicalLookback + twapLength)
+    // seconds, in blocks, for Sync events.
+    this.bufferBlockPercent = 1.1;
+
+    // TODO: Should/Can we read in `poolDecimals` from the this.uniswap?
+    this.poolPrecision = poolDecimals;
```

Comment:
nit: why do you go from calling this `Decimals` to `Precision`? Is this not a scaling factor (decimals) and not a unit of precision?

Reply:
Good point I'll rename to `poolDecimals`